src/twisted/mail/relaymanager.py

```diff
@@ -563,7 +567,7 @@ def notifySuccess(self, relay, message):
         @param message: The path of the file holding the message.
         """
         if self.noisy:
-            log.msg("success sending %s, removing from queue" % message)
+            log.msg(b"success sending " + message + b", removing from queue")
```

Comment:
I think it would be better to convert `message` to unicode  (e.g. by `repr` ?) than to emit the log text as a bytestring.